gnss/gps_time.py

```diff
@@ -92,12 +90,11 @@ def gps_minus_utc_seconds(gpst):
 
     Returns
     -------
-    utc : float
+    utc : int
       Returns the number (or an array of them) of leap second values.
     """
 
-    delta_utc = np.zeros(gpst.shape, int)
-    delta_utc = np.array(delta_utc)
+    delta_utc = np.zeros_like(gpst, int)
```

Comment:
Previously this function (and the other functions that call it) would crash if passed a `datetime.datetime` or `pd.Timestamp` object (which according to docstring should be allowed).